demo/rv/rv.env

```diff
@@ -5,4 +5,8 @@ rv_database_password=
 rv_database_port=8050
 #epid_online_url=https://verify.epid-sbx.trustedservices.intel.com/
 #epid_test_mode=false
-catalina_home=./target/tomcat
\ No newline at end of file
+catalina_home=./target/tomcat
```

Comment:
The OnDie cert cache is provided for protocol samples only. We are running smoketest using demo folder which uses component-sample binaries. The component sample rv reads the ondie cache path from .env file. There is no default value provided in the .env file, so rv dockers are running into "OnDie Initialization Error"

Also autoupdate property should be set to true by default because we are running component-samples via docker and the .cer and .crl files needs to be downloaded everytime the docker container is created.

Reply:
see previous comment